### DISCUSSION TITLE
kong fails when a child process dies

### DISCUSSION THREAD

**Discussion by kollol-chowdhury-incontact** [Kong/kong#8435] [Help]
@kikito, Please help:

Whenever a new child process is starting ,kong fails in init_worker_by_lua.
![image](https://user-images.githubusercontent.com/94824723/154925913-2f4ed3b9-687a-43f5-bf70-c5d254af7984.png)

**Comment by kikito** [comment]
Hello, that error seems to be originating in pgmoon, the library kong uses to communicate with PostgreSQL.

Which version of kong are you using? Are you able to connect to the PostgreSQL database by other means, like the psql command-line tool?

**Reply by kollol-chowdhury-incontact** [reply]
Hi @kikito,we are using kong 2.0.3.
We aren't using db to store any kong related config though.

**Reply by kikito** [reply]
The failure is related with the connection to the database. If you are not using it, I would recommend going db-less. Here are the instructions for 2.0.x (which is quite outdated at this point): https://docs.konghq.com/gateway-oss/2.0.x/db-less-and-declarative-config/
